test

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+clear
+echo "HaI"
+pwd
+find *.log>report.txt
+find *.txt>>report.txt
+tar -zcvf test.tar.gz Linux
```

Comment:
what if Linux folder contains a sample.xml and valid.out files? Would they also get tarred?